src/LibChorus/merge/xml/generic/MergeLimitedChildrenService.cs

```diff
@@ -476,11 +474,17 @@ private static XmlNode MakeClone(XmlNode node)
 
 		private static IEnumerable<XmlNode> GetElementChildren(XmlNode parent)
 		{
-			return parent == null
-					? new List<XmlNode>()
-					: (from XmlNode child in parent.ChildNodes
-					   where child.NodeType == XmlNodeType.Element // || child.NodeType == XmlNodeType.Text
-					   select child);
+			if(parent == null)
+				return new List<XmlNode>();
+
+			var elementChildren = new List<XmlNode>();
+			// Don't you dare use LINQ here, it disowns the child from the document which will break the snot out of our use of any children
```

Comment:
This is not actually true, I should revert this section of the patch. I proved myself wrong along the way